samples/fb-passthrough-sample/main.gd

```diff
@@ -0,0 +1,210 @@
+extends Node3D
+
+@export var color_map: Gradient
+@export var mono_map: Curve
+@export var brightness_contrast_saturation: Vector3
+@export var color_lut: Image
+@export var color_lut2: Image
+@export var edge_color: Color
+
+var openxr_interface: OpenXRInterface
+var fb_passthrough
+var meta_color_lut: OpenXRMetaPassthroughColorLut
+var meta_color_lut2: OpenXRMetaPassthroughColorLut
+var countdown_to_recenter_hmd: int = 3
+
+@onready var world_environment: WorldEnvironment = $WorldEnvironment
+@onready var left_controller_ray_cast: RayCast3D = $XROrigin3D/LeftController/LeftControllerRayCast
+@onready var right_controller_ray_cast: RayCast3D = $XROrigin3D/RightController/RightControllerRayCast
+@onready var passthrough_geometry: OpenXRFbPassthroughGeometry = $XROrigin3D/OpenXRFbPassthroughGeometry
+@onready var interfaces: Node3D = $Interfaces
+@onready var interface_color_map: Node3D = $Interfaces/InterfaceColorMap
+@onready var interface_mono_map: Node3D = $Interfaces/InterfaceMonoMap
+@onready var interface_brightness_contrast_saturation: Node3D = $Interfaces/InterfaceBrightnessContrastSaturation
+@onready var interface_color_map_lut: Node3D = $Interfaces/InterfaceColorMapLut
+@onready var interface_color_map_interpolated_lut: Node3D = $Interfaces/InterfaceColorMapInterpolatedLut
+@onready var interface_disabled: Node3D = $Interfaces/InterfaceDisabled
+@onready var color_map_mesh: MeshInstance3D = $Interfaces/InterfaceColorMap/ColorMapMesh
+@onready var mono_map_mesh: MeshInstance3D = $Interfaces/InterfaceMonoMap/MonoMapMesh
+
+func _ready() -> void:
+	openxr_interface = XRServer.find_interface("OpenXR")
+	if openxr_interface and openxr_interface.initialize():
+		get_viewport().use_xr = true
```

Comment:
We should also include the logic to disable vsync.

Reply:
Added!